### ISSUE TITLE
All examples are broken

### ISSUE THREAD

**Issue by coleGillespie** [sbruno81/rets-client#2]
So I am a bit confused on how this is supposed to work. When following your example I immediately get an error. 

```
TypeError: Cannot read property 'statusCode' of undefined
    at Request._callback (/home/cole/Public/github/elmstreet/node_modules/rets-client/lib/metadata.js:52:21)
    at self.callback (/home/cole/Public/github/elmstreet/node_modules/rets-client/node_modules/request/request.js:123:22)
    at Request.emit (events.js:95:17)
    at Request.self._buildRequest (/home/cole/Public/github/elmstreet/node_modules/rets-client/node_modules/request/request.js:355:39)
    at Request.init (/home/cole/Public/github/elmstreet/node_modules/rets-client/node_modules/request/request.js:533:10)
    at new Request (/home/cole/Public/github/elmstreet/node_modules/rets-client/node_modules/request/request.js:99:8)
    at request (/home/cole/Public/github/elmstreet/node_modules/rets-client/node_modules/request/index.js:54:11)
    at getMetadata (/home/cole/Public/github/elmstreet/node_modules/rets-client/lib/metadata.js:44:5)
    at Object.getResources (/home/cole/Public/github/elmstreet/node_modules/rets-client/lib/metadata.js:170:5)
    at Client.getResources (/home/cole/Public/github/elmstreet/node_modules/rets-client/index.js:193:25)
```

This is because you are trying to get the meta data on a url that has no protocol. For example you just try to get "/whatever/metadeta/" instead of padding the root uri in front of it.

For none of the examples work. How did you get this working locally? The cookies are not saving either...

**Comment by sbruno81** [comment]
Are you able to follow the login example? Are you hitting a live RETS server?

**Comment by coleGillespie** [comment]
login example works fine, also hitting a live a live rest server. the problem looks like the request inside of the metadata.js is trying to hit a url that does not exist. for example:

```javascript
    var options = {
        uri:metadataURL,
        jar:true,
        qs:{
            Type:type,
            Id:id,
            Format:format
        }
    };
```

metadataURL = "/some/metadata/path"

instead of "http://baseurl/some/metadata/path"

**Comment by dermidgen** [comment]
Yes, that is an issue with some RETS servers - it's resolved in my #develop branch. 

Regards,
Danny Graham

**Comment by sbruno81** [comment]
Ahh interesting, the RETS server I developed against returned the full URL. @dermidgen I'll incorporate your changes. Thanks!!

**Comment by sbruno81** [comment]
I went ahead and pushed a fix. The fix is in version 1.1.5. Please give it a try...
